src/index.js

```diff
@@ -0,0 +1,47 @@
+import './style.css';
+import Masonry from 'masonry-layout';
+import retrieveData from './modules/retrieveData.js';
+import Utilities from './modules/utilities.js';
+import displayData from './modules/displayData.js';
+import showLikes from './modules/showLikes.js';
+import createLike from './modules/createLike.js';
+import Jokes from './modules/jokes.js';
+import jokesCounter from './modules/jokesCounter.js';
+
+const apiBasket = 'https://v2.jokeapi.dev/joke/Programming?blacklistFlags=nsfw,racist,sexist,explicit&type=single&amount=10';
+const likesAPI = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Y7iejPnAfWtrIBxaaLI3/likes';
+
+const createLikesEvents = () => {
+  const gamesButtons = document.querySelectorAll('.likeBttns');
+  gamesButtons.forEach((bttn) => {
+    bttn.addEventListener('click', (e) => {
+      if (Utilities.hasValue(e.currentTarget.id)) {
+        createLike(likesAPI, e.currentTarget.id.substring(1));
+      }
+    });
+  });
+};
+
+const loadGrid = () => {
+  const grid = document.querySelector('.jokesGrid'); // eslint-disable-next-line
+  const masonry = new Masonry(grid, {
+    itemSelector: '.grid-item',
+    gutter: '.gutter-sizer',
+  });
+};
+
+window.addEventListener('DOMContentLoaded', () => {
+  retrieveData(apiBasket)
+    .then((response) => {
+      const { jokes } = response;
+      Jokes.setJokes(jokes);
+      displayData(jokes);
+      loadGrid();
+      createLikesEvents();
+      document.getElementById('numberJokes').innerHTML = jokesCounter();
+      retrieveData(likesAPI)
+        .then((response) => {
+          showLikes(response);
+        });
+    });
```

Comment:
**Changes Required**
- [ ] We should use `async await` syntactic sugar for the promise rather than the `.then` syntax. This will ensure that we are adhering to the ES6 syntax. Let us try to refactor this using `async await`.

Reply:
I just found how to do it 🤓 Thanks a lot for the heads-up! Changed ✔